requirements.txt

```diff
@@ -1,4 +1,5 @@
 numpy
+pywavelets==1.3.0
```

Comment:
It's a dependency of numpy. They released 1.4.0 yesterday which broke our CI, though that could just be because PyPI hadn't yet cached a `.whl` that could be downloaded.

It's possible that, now that 24 hours has passed, this is no longer needed, but it was definitely needed yesterday.

Reply:
Deleted the hard coded version/requirement. Successfully ran on CI.